archimedes/archimedes/functions/dataflow/__init__.py

```diff
@@ -1 +1,2 @@
-from .path import input_path, output_path
\ No newline at end of file
+from .path import input_path, output_path
+import json
```

Comment:
Importing `json` to mock out data for the plots, we can remove this later if it isn't really needed for the workflows.